code/game/objects/structures/window.dm

```diff
@@ -437,6 +438,7 @@
 	icon_state = "clockwork_window_single"
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	max_integrity = 100
+	armor = list(melee = 60, bullet = 25, laser = 0, energy = 0, bomb = 25, bio = 100, rad = 100, fire = 80, acid = 100)
```

Comment:
This would not be enough without the fireaxe change and I don't want brass windows to be absurdly resistant to melee more than they already are(50% melee inherited from reinforced windows)